lmdeploy/api.py

```diff
@@ -8,7 +8,6 @@
 
 
 def pipeline(model_path: str,
-             model_name: Optional[str] = None,
```

Comment:
pipeline is one of the most important APIs and this BC may affect some users when they use `model_name`. Shall we deprecate and then remove later?

Reply:
No. I already labeled this PR as BC